src/App.js

```diff
@@ -24,34 +24,36 @@ function App() {
     <>
       <LoadingProvider>
         <AuthProvider>
-          <Routes>
-            <Route path="/" element={<Layout />}>
-              {/* public routes*/}
-              <Route path="home" element={<Home />} />
-              <Route path="" element={<ExperimentalHome />} />
-              <Route path="login" element={<Login />} />
-              <Route path="register" element={<Register />} />
-              <Route path="team" element={<Team />} />
-              <Route path="events" element={<Events />} />
-              <Route path="get_started" element={<GetStarted />} />
+          <BrowserRouter>
+            <Routes>
+              <Route path="/" element={<Layout />}>
+                {/* public routes*/}
+                <Route path="home" element={<Home />} />
+                <Route path="" element={<ExperimentalHome />} />
+                <Route path="login" element={<Login />} />
+                <Route path="register" element={<Register />} />
+                <Route path="team" element={<Team />} />
+                <Route path="events" element={<Events />} />
+                <Route path="get_started" element={<GetStarted />} />
 
-              {/* private routes */}
-              <Route element={<RequireAuth />}>
-                <Route path="forum" element={<AskREC />} />
-                <Route path="blog" element={<Blog />} />
-                <Route path="experience" element={<Experiences />} />
-                <Route
-                  path="experience/detail/:experienceId"
-                  element={<DetailExperience />}
-                />
+                {/* private routes */}
+                {/* <Route element={<RequireAuth />}>
+                  <Route path="forum" element={<AskREC />} />
+                  <Route path="blog" element={<Blog />} />
+                  <Route path="experience" element={<Experiences />} />
+                  <Route
+                    path="experience/detail/:experienceId"
+                    element={<DetailExperience />}
+                  />
+                </Route> */}
+                {/* <Route
```

Comment:
Please don't comment out code. It may affect other modules. If you are commenting out something to test , please dont commit those changes.